utils/fetchArticles.js

```diff
@@ -1,12 +1,13 @@
 import axios from "axios";
 
-let url = "https://zia-nc-news.onrender.com/api/articles";
-
 export default function fetchArticles(articleId) {
-  if (articleId === undefined) {
-    return axios.get(url).then((articles) => {
-      //   console.log(articles.data);
-      return articles.data;
-    });
+  let url = "https://zia-nc-news.onrender.com/api/articles";
+
+  if (articleId !== undefined) {
+    url += `/${articleId}`;
   }
+
+  return axios.get(url).then((articles) => {
+    return articles.data;
+  });
 }
```

Comment:
It might be worth considering splitting these out into 2 functions:

One which fetches all articles

One which fetches articles by ID